src/tests/presets/test_account.cairo

```diff
@@ -133,78 +134,72 @@ fn is_valid_sig_dispatcher() -> (AccountABIDispatcher, felt252, Array<felt252>)
 }
 
 #[test]
-#[available_gas(2000000)]
 fn test_is_valid_signature() {
     let (dispatcher, hash, signature) = is_valid_sig_dispatcher();
 
     let is_valid = dispatcher.is_valid_signature(hash, signature);
-    assert(is_valid == starknet::VALIDATED, 'Should accept valid signature');
+    assert_eq!(is_valid, starknet::VALIDATED);
 }
 
 #[test]
-#[available_gas(2000000)]
 fn test_is_valid_signature_bad_sig() {
     let (dispatcher, hash, _) = is_valid_sig_dispatcher();
 
     let bad_signature = array![0x987, 0x564];
 
     let is_valid = dispatcher.is_valid_signature(hash, bad_signature.clone());
-    assert(is_valid == 0, 'Should reject invalid signature');
+    assert!(is_valid.is_zero(), "Should reject invalid signature");
 }
 
 #[test]
-#[available_gas(2000000)]
 fn test_isValidSignature() {
     let (dispatcher, hash, signature) = is_valid_sig_dispatcher();
 
     let is_valid = dispatcher.isValidSignature(hash, signature);
-    assert(is_valid == starknet::VALIDATED, 'Should accept valid signature');
+    assert_eq!(is_valid, starknet::VALIDATED);
 }
 
 #[test]
-#[available_gas(2000000)]
 fn test_isValidSignature_bad_sig() {
     let (dispatcher, hash, _) = is_valid_sig_dispatcher();
 
     let bad_signature = array![0x987, 0x564];
 
     let is_valid = dispatcher.isValidSignature(hash, bad_signature);
-    assert(is_valid == 0, 'Should reject invalid signature');
+    assert!(is_valid.is_zero(), "Should reject invalid signature");
 }
 
 //
 // supports_interface
 //
 
 #[test]
-#[available_gas(2000000)]
 fn test_supports_interface() {
     let dispatcher = setup_dispatcher();
-    assert(dispatcher.supports_interface(ISRC5_ID), 'Should implement ISRC5');
-    assert(dispatcher.supports_interface(ISRC6_ID), 'Should implement ISRC6');
-    assert(!dispatcher.supports_interface(0x123), 'Should not implement 0x123');
+    let supports_isrc5 = dispatcher.supports_interface(ISRC5_ID);
+    let supports_isrc6 = dispatcher.supports_interface(ISRC6_ID);
+    let doesnt_support_0x123 = !dispatcher.supports_interface(0x123);
+    assert!(supports_isrc5);
+    assert!(supports_isrc6);
+    assert!(doesnt_support_0x123);
```

Comment:
we should settle on a query/assert order pattern. in here it's all queries then asserts, in the previous test it's mixed, and there's also query/assert pairs

Reply:
Asserting right after each query as you suggested looks like the best approach to me.

Reply:
added a comment in [the style guidelines issue](https://github.com/OpenZeppelin/cairo-contracts/issues/696#issuecomment-1915511208)